decidim-admin/app/queries/decidim/admin/moderation_stats_query.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class ModerationStatsQuery
```

Comment:
Just a minor cosmetic change, but as far as I see we don't usually add the "query" suffix on other Query classes (`find decidim-*/app/queries/ -type f -name "*query*"` only gives me this file). Can we drop it?  
```suggestion
    class ModerationStats
```